Modules/Loadable/VolumeRendering/MRML/vtkMRMLVolumeRenderingDisplayNode.h

```diff
@@ -157,6 +162,8 @@ class VTK_SLICER_VOLUMERENDERING_MODULE_MRML_EXPORT vtkMRMLVolumeRenderingDispla
 
   int CroppingEnabled;
 
+  int ROIDisplayEnabled;
```

Comment:
I have done (see last commit)

in practice I used this in the slot of qSlicerVolumeRenderingModuleWidget for enabling the visibility:

d->ROIWidget->mrmlROINode()->SetDisplayVisibility(toggle);
d->ROIWidget->mrmlROINode()->Modified();